packages/paste-core/components/meter/src/Meter.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import {type BoxProps, Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import type {HTMLPasteProps} from '@twilio-paste/types';
+import {useMeter} from '@twilio-paste/react-spectrum-library';
+
+export const labelIdSuffix = 'LABEL';
+
+export interface MeterProps extends HTMLPasteProps<'meter'>, Pick<BoxProps, 'element'> {
+  minValue?: number;
+  maxValue?: number;
+  value?: number;
+  id: string;
+  showValueLabel?: boolean;
+  formatOptions?: Intl.NumberFormatOptions; // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#options
+  valueLabel?: string;
+  /*
+   * The following props don't exist on the react-aria useMeter hook but do exist on the HTML meter element.
+   * They can be added back into the Paste Meter API depending on the finalized spec & designs.
+   *
+   * low?: number;
+   * high?: number;
+   * optimum?: number;
+   */
+}
+
+const Meter = React.forwardRef<HTMLMeterElement, MeterProps>(({element = 'METER', id, ...props}, ref) => {
+  const {value = 0, minValue = 0, maxValue = 100, showValueLabel = true} = props;
+  const {meterProps} = useMeter(props);
+
+  // Calculate the width of the bar as a percentage
+  const percentage = (value - minValue) / (maxValue - minValue);
+  const fillWidth = `${Math.round(percentage * 100)}%`;
+
+  return (
+    <Box
+      as="div"
+      {...meterProps}
+      role="meter"
+      id={id}
+      maxWidth="size30"
+      position="relative"
+      element={element}
+      aria-labelledby={`${id}${labelIdSuffix}`}
```

Comment:
thought: by making `id` required and doing this like this, we're effectively saying you cannot label a Meter with `aria-label` or a custom `aria-labelledby`. Do we still want to allow custom values for those labeling methods?

Reply:
updated to allow custom labeling options